Lesson3/Task5.java

```diff
@@ -0,0 +1,11 @@
+public class Task5 {
+	public static void main(String[] args) {
+		String a = "hop";
+		String b = "hello";
+		if (a.length() < b.length()) 
+		 System.out.println(a.concat(b.concat(a)));
```

Comment:
This one is good but would be better if you use methods which were on the lesson, like String.join.